code/modules/antag_contest/contest_verbs.dm

```diff
@@ -0,0 +1,225 @@
+/*
+ * Client verbs for the antag contest. Disable this file when the contest is over!
+ * AUG2016
+ */
+
+ #define INDEP		0
+ #define PRO_SYNTH	1
+ #define ANTI_SYNTH	2
+
+// Displays a simple little guide.
+/client/verb/contest_help()
+	set name = "Contest Help"
+	set category = "Contest"
+	set desc = "Information about the contest."
+
+	if (!config.antag_contest_enabled)
+		src << "<span class='warning'>The contest isn't running yet!</span>"
+		return
+
+	var/help = file2text('ingame_manuals/antag_contest.html')
+
+	if (!help)
+		src << "<span class='warning'>Unable to open the document required! Please contact administration or a coder!</span>"
+		return
+
+	src << browse(help, "window=antag_contest_help;size=600x500")
+
+/client/verb/contest_my_characters()
+	set name = "My Character Status"
+	set category = "Contest"
+	set desc = "Displays information to you about your characters, and their standings."
+
+	if (!config.antag_contest_enabled)
+		src << "<span class='warning'>The contest isn't running yet!</span>"
+		return
+
+	if (!establish_db_connection(dbcon))
+		src << "<span class='warning'>Failed to establish SQL connection! Contact a member of staff!</span>"
+		return
+
+	var/DBQuery/character_query = dbcon.NewQuery("SELECT id, name FROM ss13_characters WHERE ckey = :ckey AND deleted_at IS NULL")
+	character_query.Execute(":ckey" = src.ckey)
```

Comment:
This is a mistake. Will be fixed. 2 other Execute statements were pass kwargs and not lists.
I should maybe make it use kwargs. HRM. In the future, maybe.